README.md

```diff
@@ -166,16 +178,69 @@ Change the model in the ask command.
 ```bash
 pieces change_model [MODEL_INDEX]
 ```
+##### Conversations command
+###### Coversations
+
+List all conversations that you have. The green conversation is the one that is currently being used in the ask command
+
+```bash
+pieces conversations
+```
+
+###### Conversation
+
+List the messages in the currently using conversation in the ask command. 
+
+```bash
+pieces conversation
+```
+
+You can create a new conversation that will be used in the ask command.
+
+```bash
+pieces conversation -n
+```
+
+Rename a conversation
+
+```bash
+pieces conversation -r My awsome name
+```
+
+Or you can make the model rename it for you
+
+```bash
+pieces conversation -r
+```
+
+Delete a conversation
+
+```bash
+pieces conversation -d
+```
+
+
+
+You can switch the conversation and list its messages. Check the conversations command to get the index
+
+```bash
+pieces conversation x
+```
+
+
 ##### Ask a question to a model:
-** Requires quotes around question **
+**Requires quotes around question**
 
-Ask the copoilt a question it uses chatGPT3 as a defualt model to ask a question, you can change the model using the change model command.
+Ask the copoilt a question it uses chatGPT3 as a defualt model to ask a question, you can change the model using the change model command. You can add a relevance file or snippet index check the list assets command.
+You can use `.` or `/` to refer to the current directory 
 
 ```bash
-pieces ask "your question"
+pieces ask "your question" -f /file1 /file2 folder -s 1 2 3
```

Comment:
Interesting, so you're asking a question of a specific file?

Reply:
Or folder

Reply:
Very cool